app/src/ui/lib/update-store.ts

```diff
@@ -182,11 +189,13 @@ class UpdateStore {
 
     this.userInitiatedUpdate = !inBackground
 
-    try {
-      autoUpdater.setFeedURL({ url: updatesURL })
-      autoUpdater.checkForUpdates()
-    } catch (e) {
-      this.emitError(e)
+    const checkedResults = await ipcRenderer.invoke(
+      'check-for-updates',
+      updatesURL
+    )
+
+    if (checkedResults !== null && checkedResults !== undefined) {
+      this.emitError(checkedResults)
     }
```

Comment:
Probably worth renaming this to `error`?
```suggestion
    const error = await ipcRenderer.invoke(
      'check-for-updates',
      updatesURL
    )

    if (error !== null && error !== undefined) {
      this.emitError(error)
    }
```